tutorial/godiva/godiva_openmc_1200.py

```diff
@@ -21,7 +21,7 @@
 geom.export_to_xml()
 
 settings = openmc.Settings()
-settings.batches = 200
+settings.batches = 100
```

Comment:
Is this change to save time?

Reply:
I'm not sure why the simulation at 900K used 100 batches while 1200K used 200 batches. It seems like a mistake so I wanted to standardize both to 100 batches.